src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js

```diff
@@ -0,0 +1,175 @@
+import { useState } from 'react';
+import { InjectAppServices } from '../../../../../services/pure-di';
+import { useIntl, FormattedMessage } from 'react-intl';
+import {
+  FieldGroup,
+  FieldItem,
+  InputFieldItem,
+  PhoneFieldItem,
+  EmailFieldItem,
+  SubmitButton,
+} from '../../../../form-helpers/form-helpers';
+import { Form, Formik } from 'formik';
+import { getFormInitialValues } from '../../../../../utils';
+import useTimeout from '../../../../../hooks/useTimeout';
+import { useNavigate } from 'react-router-dom';
+
+const fieldNames = {
+  firstname: 'firstname',
+  lastname: 'lastname',
+  email: 'email',
+  phone: 'phone',
+};
+
+const displayBlockStyle = {
+  display: 'block',
+};
+
+export const DELAY_BEFORE_REDIRECT_TO_LOGIN = 3000;
+
+const HAS_ERROR = 'HAS_ERROR';
+const SAVED = 'SAVED';
+
+export const ContactInformation = InjectAppServices(
+  ({ dependencies: { dopplerBillingUserApiClient } }) => {
+    const intl = useIntl();
+    const _ = (id, values) => intl.formatMessage({ id: id }, values);
+    const createTimeout = useTimeout();
+    const navigate = useNavigate();
+    const [status, setStatus] = useState('');
+
+    const _getFormInitialValues = () => {
+      return getFormInitialValues(fieldNames);
+    };
+
+    const sendContactInformation = async (values) => {
+      const result = await dopplerBillingUserApiClient.sendContactInformation(values);
+      if (result.success) {
+        setStatus(SAVED);
+
+        createTimeout(() => {
+          navigate('/login');
+        }, DELAY_BEFORE_REDIRECT_TO_LOGIN);
+      } else {
+        setStatus(HAS_ERROR);
+      }
+    };
```

Comment:
can you add some test for to verify when the form is submited and the user is redirect?